packages/xrpl/test/models/offerCreate.test.ts

```diff
@@ -86,6 +89,111 @@ describe('OfferCreate', function () {
     assertValid(offer3)
   })
 
+  it(`verfies valid offerCreate within permissioned domain`, function () {
+    const offer = {
```

Comment:
```suggestion
    const offerTx = {
```
Just to be consistent with naming transactions. Ditto for the remaining ones below this test too.

Reply:
75ae1c3d7